tajo-catalog/tajo-catalog-server/src/main/resources/schemas/mariadb/mariadb.xml

```diff
@@ -19,6 +19,7 @@
 <tns:store xmlns:tns="http://tajo.apache.org/catalogstore" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://tajo.apache.org/catalogstore ../DBMSSchemaDefinition.xsd ">
   <!--
       Catalog base version history
+      * 13 - 2016-05-07: Added general index consist of DB_ID, TID, COLUMN_NAMES (TAJO-2117)
```

Comment:
You need to use the title of origin JIRA issue. If you want to write other comments to above history, you should rename the title of JIRA issue.